### DISCUSSION TITLE
cron job and job shut down

### DISCUSSION THREAD

**Discussion by saba-bakhshi** [linkerd/linkerd2#6642] [Q&A]
is there any way to terminate meshed jobs and cron jobs without going through the manual process? 
I know I can kill the proxy using curl  -XPOST localhost:4191/shutdown, but can we add something to the annotation instead so that it's not manual?

**Comment by alpeb** [comment]
There's no direct way to do this via an annotation. However you can wrap your job's process around a `linkerd-await` call, which will shutdown the proxy when the process is done, as exemplified in [this Dockerfile](https://github.com/linkerd/linkerd-await#dockerfile).
